OWNERS_ALIASES

```diff
@@ -45,6 +45,7 @@ aliases:
     - palnabarun
   sig-docs-leads:
     - divya-mohan0209
+    - katcosgrove
     - kbhawkey
     - natalisucks
     - onlydole
```

Comment:
Can be done in a follow up - but should sync thing with current state kbhawkey and onlydole should be removed and everyone else added^^;;

/lgtm
/approve